x-pack/legacy/plugins/apm/public/components/app/ErrorGroupDetails/DetailView/__snapshots__/index.test.tsx.snap

```diff
@@ -36,7 +36,7 @@ exports[`DetailView should render Discover button 1`] = `
   <EuiButtonEmpty
     iconType="discoverApp"
   >
-    View 10 occurrences in Discover
+    View 10 occurences in Discover.
```

Comment:
Isn't "occurrences" with 2 r's?

Reply:
Good catch. I copy/pasted from the ticket.

Reply:
Fixed in 10c6f3b0ed1.